easybuild/easyconfigs/d/DOLFINx-MPC/DOLFINx-MPC-0.9.0-foss-2023b.eb

```diff
@@ -0,0 +1,54 @@
+# Authors:: Tanmoy Chakraborty (University of Warwick/tanmoy.chakraborty@warwick.ac.uk)
+
+easyblock = 'CMakePythonPackage'
+
+name = 'DOLFINx-MPC'
+version = '0.9.0'
+
+homepage = 'https://github.com/jorgensd/dolfinx_mpc'
+description = "Implementation of multi-point constraints in DOLFINx."
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+source_urls = ['https://github.com/jorgensd/dolfinx_mpc/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['9b87cd75e3b26b3f92b600e935c45677624c8e169782bf6646265d1e03a0e043']
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+    ('pkgconf', '2.0.3'),
+    ('Ninja', '1.11.1'),
+    ('Python', '3.11.5'),
+    ('scikit-build-core', '0.10.7'),
+]
+
+dependencies = [
+    ('PETSc', '3.20.3'),
+    ('Eigen', '3.4.0'),
+    ('petsc4py', '3.20.3'),
```

Comment:
Maybe change to the already merged `petsc4py/3.22.5`, which is currently on GitHub, so we can finish that PR?